creme/creme_core/static/creme_core/js/lv_widget.js

```diff
@@ -35,6 +35,33 @@ creme.lv_widget.checkSelectionMode = function(mode) {
     }
 };
 
+creme.lv_widget.DownloadAction = creme.component.Action.sub({
```

Comment:
I think ExportAction would be better ("download" is confusing with downloading of Document -- there is no backend choices -- for example).